src/python/espressomd/lb.pyx

```diff
@@ -542,3 +548,144 @@ cdef class LBFluidRoutines:
+                        np.array([x, y, z])), prop_name)
+        return res
+
+    def set_values(self, _x_indices, _y_indices, _z_indices, prop_name, value):
+        for i, x in enumerate(_x_indices):
+            for j, y in enumerate(_y_indices):
+                for k, z in enumerate(_z_indices):
+                    setattr(LBFluidRoutines(
+                        np.array([x, y, z])), prop_name, value[i, j, k])
+
+    property density:
+        def __get__(self):
+            prop_name = "density"
+            shape_res = (1,)
+            return np.squeeze(self.get_values(
+                self.x_indices, self.y_indices, self.z_indices, prop_name, shape_res), axis=3)
+
+        def __set__(self, value):
+            prop_name = "density"
+            self.set_values(
+                self.x_indices,
+                self.y_indices,
+                self.z_indices,
+                prop_name,
+                value)
+
+    property index:
+        def __get__(self):
+            prop_name = "index"
+            shape_res = (3,)
+            return self.get_values(
+                self.x_indices, self.y_indices, self.z_indices, prop_name, shape_res)
+
+    property velocity:
+        def __get__(self):
+            prop_name = "velocity"
+            shape_res = (3,)
+            return self.get_values(
+                self.x_indices, self.y_indices, self.z_indices, prop_name, shape_res)
+
+        def __set__(self, value):
+            prop_name = "velocity"
+            if value.shape == (len(self.x_indices), len(
+                    self.y_indices), len(self.z_indices), 3):
+                self.set_values(
+                    self.x_indices,
+                    self.y_indices,
+                    self.z_indices,
+                    prop_name,
+                    value)
+            else:
+                raise ValueError(
+                    "Input-dimensions of velocity array", value.shape, "does not match slice dimensions",
+                    (len(self.x_indices), len(self.y_indices), len(self.z_indices), 3), ".")
+
+    property pressure_tensor:
+        def __get__(self):
+            prop_name = "pressure_tensor"
+            shape_res = (3, 3)
+
+            return self.get_values(
+                self.x_indices, self.y_indices, self.z_indices, prop_name, shape_res)
+
+        def __set__(self, value):
+            raise NotImplementedError
+
+    property pressure_tensor_neq:
+        def __get__(self):
+            prop_name = "pressure_tensor_neq"
+            shape_res = (3, 3)
+
+            return self.get_values(
+                self.x_indices, self.y_indices, self.z_indices, prop_name, shape_res)
+
+        def __set__(self, value):
+            raise NotImplementedError
+
+    property population:
+        def __get__(self):
+            prop_name = "population"
+            shape_res = (19,)
+            cdef Vector19d double_return
+            double_return = lb_lbnode_get_pop(self.node)
```

Comment:
this is not needed since the populations are interfaced in the ```LBFluid```

Reply:
true, will be fixed